render/container.go

```diff
@@ -177,20 +169,9 @@ var ContainerImageRenderer = Memoise(FilterEmpty(report.Container,
 //
 // not memoised
 var ContainerHostnameRenderer = FilterEmpty(report.Container,
-	MakeReduce(
-		MakeMap(
-			MapContainer2Hostname,
-			ContainerWithImageNameRenderer,
-		),
-		// Grab *all* the hostnames, so we can count the number which were empty
-		// for accurate stats.
-		MakeMap(
-			MapToEmpty,
```

Comment:
After removing it from here, this method is not used anymore so feel free to remove it from `container.go`.